pkg/kudoctl/cmd/env/kudohome/home.go

```diff
@@ -0,0 +1,31 @@
+package kudohome
+
+import (
+	"os"
+	"path/filepath"
+)
+
+// Home describes the location of a CLI configuration.
+type Home string
+
+// String returns Home as a string with expansions $HOME == home
+func (h Home) String() string {
+	return os.ExpandEnv(string(h))
+}
+
+// Path returns Home with elements appended.
+func (h Home) Path(elem ...string) string {
+	p := []string{h.String()}
+	p = append(p, elem...)
+	return filepath.Join(p...)
+}
+
+// Repository returns the path to the local repository.
+func (h Home) Repository() string {
+	return h.Path("repository")
+}
+
+// RepositoryFile returns the path to the repositories.yaml file.
+func (h Home) RepositoryFile() string {
+	return h.Path("repository", "repositories.yaml")
```

Comment:
why are repository and repo file just constants somewhere? I don't think we need to dynamically construct those...

Reply:
* allows home to be typed with paths relative to home
* allows os independence with filepath joins ensure proper form
* allows for ENV expansion so home can be set with `--home=$HOME/kudo2` which will expand $HOME to user home and will perform above mentioned benefits.